src/index.html

```diff
@@ -3,11 +3,236 @@
+            </p>
+          </div>
+
+          <div class="section__details">
+            <div class="section__details-img section__details-img--5 section__details-img--long section__details-img--special">
+              <img
+                class="visually-hidden"
+                src="/images/details/5.png"
+                alt="Bike with hydraulic brakes"
+              >
+            </div>
+            <div class="section__details-img section__details-img--special-long section__details-img--6">
+              <img
+                class="visually-hidden"
+                src="/images/details/6.png"
+                alt="Bicycle that has hydraulic disk brakes image - 2"
+              >
+            </div>
+            <h4 class="section__element-title section__element-title--details">Hydraulic disc brakes & Lightweight</h4>
+            <p class="section__description section__description--details">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </div>
+
+          <a href="#" class="button section__button button--324">Explore</a>
+        </section>
+
+        <section class="section main__section main__section--m-biggest">
+          <h2 class="section__title" id="contact">Contact us</h2>
+
+          <div class="section__details">
+            <div class="form">
+              <form
+                class="form__content"
+                action="/"
+                method="post"
+                id="landing-form"
+              >
+                <label for="name" class="form__label"></label>
+                <input
+                  class="input form__name"
+                  type="text" id="name"
+                  placeholder="Name"
+                  required
+                >
+                <label for="email" class="form__label"></label>
+                <input
+                  class="input form__email"
+                  type="email"
+                  id="email"
+                  placeholder="Email"
+                  required
+                >
+                <textarea class="form__text" placeholder="Message" required
+                ></textarea>
+                <button class="button form__button" type="submit">
+                  Submit
+                </button>
+              </form>
+            </div>
+
+            <div class="address section__address">
+              <p class="address__title">Phone</p>
+              <p class="address__content">+1 234 5555-55-55</p>
+
+              <p class="address__title">Email</p>
+              <p class="address__content">hello@maiami.com</p>
+
+              <p class="address__title">Address</p>
+              <p class="address__content">400 first ave.<br>suite 700<br> Minneapolis, MN 55401</p>
```

Comment:
would be awesome to make
click on phone - trigger a calling action on device (`<a href="tel:...`)
email - open email app (`<a href = "mailto:...`)
location: `<a href="https://goo.gl/maps`

plus
> 9.	Location-related addresses / links / images should open google maps in a new tab target="_blank"

Reply:
Good point. Fixed.